kuma/users/tests/test_views.py

```diff
@@ -586,6 +612,9 @@ def test_persona_signup_create_django_user(self):
         persona_signup_email = 'views_persona_django_user@example.com'
         persona_signup_username = 'views_persona_django_user'
 
+        # Put recaptcha into testing mode.
```

Comment:
You can use the test decorator `@mock.patch.dict(os.environ, {'RECAPTCHA_TESTING': 'true'})` for this